clang/test/AST/ByteCode/builtins.cpp

```diff
@@ -36,3 +36,10 @@ void test_builtin_os_log(void *buf, int i, const char *data) {
   constexpr int len = __builtin_os_log_format_buffer_size("%d %{public}s %{private}.16P", i, data, data);
   static_assert(len > 0, "Expect len > 0");
 }
+
+void addc_dummy()
+{
+  unsigned int a;
+  if (__builtin_addc (1, 42, 0, &a))
```

Comment:
What makes `&a` a dummy pointer here? I was looking at the code trying to understand the meaning of dummy but it was not clear.